hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/AutoAccountCreationSuite.java

```diff
@@ -1178,6 +1266,143 @@ private HapiSpec hollowAccountCompletionWithContractCall() {
                                 }));
     }
 
+    private HapiSpec hollowAccountCompletionViaSignatureInSigMap() {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Here we mean to test the case in which the payer is not a hollow account?

Reply:
Hmm, I think this test case is no longer present in the final code